<p>PLM rule to avoid trigger off unless mandatory and trigg on if trigg on is used</p>
<h2>Noncompliant Code Example</h2>
<pre>
			MqlUtil.mqlCommand(context,"trigg off");
</pre>
<h2>Compliant Solution</h2>
<pre>
Compliant code solution for this is scenario not available please avoid using this as much as possible
</pre>
