<p>PLM rule to Avoid hardcoding for http/https</p>
<h2>Noncompliant Code Example</h2>
<pre>
Do not hardcode URL calls
</pre>
<h2>Compliant Solution</h2>
<pre>
use propertyfile or Configuration Object
</pre>
